### DISCUSSION TITLE
Can I make a rebrand?

### DISCUSSION THREAD

**Discussion by tripod3d** [picotorrent/picotorrent#1188] [General]
Hi,

There is a company which would download and share a lot of data between offices around the states and the best option was to use P2P via bittorrent. 

So I was wondering if can I make a custom version of this client, because is tiny and fast, but hiding some options I don't want to be modified as it will be preconfigured to match the network already installed.

Also hiding the fact that it is a torrent client so coworkers can't download anything else on the internet. Plus some company brand will prevent this.

How can I hide some elements? How can I rebrand? I also can figure it my self, but most important and where I really need help is how can I compile it as a portable like your release, that it is tiny and fast.

Thank you a lot.

**Comment by ghost** [comment]
you'll have to check the licenses. if they allow it, then sure, you can change it and redistribute it. regarding gui modifications, this program was made with wxwidgets, so you'll have to use that to modify it.
